src/Compilers/Core/MSBuildTask/project.json

```diff
@@ -12,6 +12,7 @@
         "System.IO.FileSystem": "4.0.1",
         "System.IO.FileSystem.DriveInfo": "4.0.0",
         "System.IO.Pipes": "4.0.0",
+        "System.Reflection": "4.1.0",
```

Comment:
Does this get correctly bundled in the compiler extension? I'd hope so but it's hard to verify.